### PR TITLE
Black puzzles list

### PR DESCRIPTION
**Pull request by l4ur4g10**
In this PR I want to contribute adding a list of the black puzzles.

### CODE REVIEW

File: puzzle.py [hunk 1]
```diff
@@ -149,57 +149,224 @@ def print_puzzles(puzzles: typing.Sequence[typing.Optional[Puzzle]]) -> None:
 black_puzzles = [
     Puzzle(
         [
-            [1, 1, 0, 1, 1],
+            [1, 1, 1, 1, 1],
+            [1, 1, 1, 0, 1],
+            [1, 1, 0, 0, 1]
+            [0, 0, 0, 0, 1]
```
Comment (cl3t0) [inline puzzle.py]: Missing commas in these two lines.

File: puzzle.py [hunk 2]
```diff
@@ -149,57 +149,224 @@ def print_puzzles(puzzles: typing.Sequence[typing.Optional[Puzzle]]) -> None:
...
+    ),
+    Puzzle([
+        [1, 1, 1, 1, 1],
+        [1, 1, 1, 1, 1],
+        [1, 0, 0, 1, 1],
+        [0, 0, 0, 1, 1],
+        [0, 0, 0, 0, 1]
+    ],
+    3,
+    Piece.CORNER
```
Comment (cl3t0) [inline puzzle.py]: Need code formatting.

**Review by cl3t0 — CHANGES_REQUESTED** [review]

File: puzzle.py (hunk 2 above)
Comment (l4ur4g10) [reply puzzle.py]: Fixed

File: puzzle.py (hunk 1 above)
Comment (l4ur4g10) [reply puzzle.py]: Fixed

**Review by cl3t0 — APPROVED** [review]

**Pull request merged**